packages/accounts/tests/client/accounts_test.coffee

```diff
@@ -0,0 +1,62 @@
+describe("Account header", () ->
+
+  beforeEach((test) ->
+    $('.modal').modal('hide')
+    Meteor.call("clearUsers")
```

Comment:
Do you mean wrap the rest of the test setup in a waitFor and put a setTimeout in the callback of clearUsers?